subprojects/core/src/integTest/groovy/org/gradle/api/tasks/IncrementalBuildIntegrationTest.groovy

```diff
@@ -1547,7 +1547,7 @@ task b(dependsOn: a)
             ignoredAnnotatedPropertyMessage {
                 type('CustomTask').property('classpath')
                 ignoring('Internal')
-                alsoAnnotatedWith('InputFiles', 'Classpath')
+                alsoAnnotatedWith('Classpath', 'InputFiles')
```

Comment:
Why does the order of the classes change here? Is this related to the use of `LinkedHashMultimap` instead of `HashMultimap`?

Reply:
It seems that something has changed about the iteration order of an `ArrayListMultimap` between versions. I made the ordering more explicit now.